src/BaseTemplates/StarterWeb/Startup.cs

```diff
@@ -19,10 +19,14 @@ public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
             // Set up configuration sources.
             var builder = new ConfigurationBuilder()
                 .SetBasePath(appEnv.ApplicationBasePath)
-                .AddJsonFile("appsettings.json")
+                .AddJsonFile(source =>
+                {
+                    source.Path = "appsettings.json";
+                    source.ReloadOnChange = true;
```

Comment:
Didn't we create a simple overload for this? Does the simplest/most common usage in our default template require a lambda for this provider while it doesn't for the others? :disappointed: